### PR TITLE
feat: Added List

### PR DESCRIPTION
**Pull request by parthgabani777**
Added two variants of the list: - 
- simple list
- notification list with image and dismiss button

### CODE REVIEW

**Review by devananth — COMMENTED** [review]
Everything is fine , code looks good !

File: docs.css [hunk 1]
```diff
@@ -51,6 +51,7 @@ header{
 }
 .code-snippet iframe{
       margin-bottom:2rem;
+      margin-top:2rem;
```
Comment (Prabhav-Sharma) [inline docs.css]: Use global variables here as well.

**Review by Prabhav-Sharma — COMMENTED** [review]
Everything else is good.

File: docs.css (hunk 1 above)
Comment (parthgabani777) [reply docs.css]: ok.

**Pull request merged**